pywbem/_logging.py

```diff
@@ -48,8 +48,25 @@
   responses between the pywbem client and WBEM server. This logs the HTTP
   request data and response data including HTTP headers and CIM-XML payload.
 
-These pywbem loggers output log records when they are configured and activated.
-The pywbem loggers log at the :attr:`py:logging.DEBUG` logging level.
+Pywbem uses the :attr:`py:logging.DEBUG` logging level for both loggers.
+
+The best practices for Python libraries suggest that libraries such as pywbem
+have a null handler attached to the Python logger for the package (the
+logger named `'pywbem'`, in this case), because since Python 2.7, the Python
+root logger will by default (i.e. when not being configured) print log
+records of logging level :attr:`py:logging.WARNING` or greater to `sys.stderr`.
+Having a null handler on the logger for the library prevent such log events
+to propagate up to the Python root logger, and thus the library will be silent
```

Comment:
Change "log events to propagate" to "log events from propagating". Don't you just love english..

You can also remove the word thus in "thus the library will be slient"

Reply:
DONE, both.